plugin/proxy/README.md

```diff
@@ -64,6 +64,8 @@ There are three load-balancing policies available:
 * `random` (default) - Randomly select a backend
 * `least_conn` - Select the backend with the fewest active connections
 * `round_robin` - Select the backend in round-robin fashion
+* `first` - Select the first available backend looking by order of declaration from left to right
+
```

Comment:
Minor note - it should be "There are **four** load-balancing policies available:" at line 63.

Reply:
you are right :)